content/sensu-core/1.4/guides/monitor-the-monitor.md

```diff
@@ -0,0 +1,197 @@
+
+### Monitoring Uchiwa{#monitoring-uchiwa-dashboard}
+
+*Method 1: Monitor Uchiwa with a process check*
+
+To monitor the Uchiwa Dashboard, you will need to check for two processes named `uchiwa` using [check-process sensu plugin][9] using the following check definition. This check will return a check result with status 2 if less than 2 processes are running with the string `/opt/uchiwa/bin/uchiwa`. We look for two as there is a parent and child process for the `uchiwa service`
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_uchiwa_process": {
+      "command": "/opt/sensu/embedded/bin/check-process.rb -p /opt/uchiwa/bin/uchiwa -C 2",
+      "subscribers": [
+        "monitor_local_uchiwa_processes"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+*Method 2: Monitor Uchiwa with a remote network port check*
+
+You can also monitor the `uchiwa` Dashboard with a remote port check using the [check-port sensu plugin][8] with the following check definition.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_uchiwa_port": {
+      "command": "check-ports.rb -H uchiwa-remote-hostname -p 3000",
+      "subscribers": [
+        "monitor_remote_uchiwa_port"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+_PRO TIP: Use both checks for complete monitoring of your Uchiwa dashboard: catch when both processes are not running and when a firewall port is not open_
+
+## Monitoring RabbitMQ{#monitoring-rabbitmq}
+
+To monitor that your RabbitMQ instance is responding, you will need to do so from an independent Sensu stack. The [rabbitmq-alive sensu plugin][10] provides that ability using the below configuration with your RabbitMQ credentials.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_rabbitmq_alive": {
+      "command": "check-rabbitmq-alive.rb -w remote-rabbitmq-host -v remote-rabbitmq-vhost -u remote-rabbitmq-username -p remote-rabbitmq-password -P remote-rabbitmq-listen-port",
+      "subscribers": [
+        "monitor_remote_rabbitmq"
+      ],
+      "interval": 60
+    }
+  }
+}
+{{< /highlight >}}
+
+While Sensu does not provide benchmarks for a healthy RabbitMQ queue, you can trend the data using the [metrics-rabbitmq-queue sensu plugin][11] metrics check for your instances. In the below check definition, metrics collection is done on the same host as RabbitMQ lives.
+
+{{< highlight json >}}
+{
+  "checks": {
+    "collect_rabbitmq_queue": {
+      "command": "metrics-rabbitmq-queue.rb --host localhost --password :::rabbitmq.password::: --port 15672 --user sensu --filter keepalives",
+      "subscribers": [
+        "rabbitmq"
+      ],
+      "interval": 60,
+      "type": "metric"
+    }
+  }
+}
+{{< /highlight >}}
+
+then make a check using the [check-rabbitmq-check sensu plugin][12] to then provide an alert when the queue starts to fill up using the following definition
+
+{{< highlight json >}}
+{
+  "checks": {
+    "check_rabbitmq_queue": {
+      "command": "TODO",
```

Comment:
Looks like the command is missing here

Reply:
Struggling to get a working command here for some reason. I'll add it once I got it working, but all the other requested changes have been done.